org.jacoco.core.test/src/org/jacoco/core/internal/instr/InstrSupportTest.java

```diff
@@ -56,17 +56,17 @@ public void needFrames_should_return_true_for_versions_greater_than_or_equal_to_
 	}
 
 	@Test
-	public void testAssertNotIntrumentedPositive() {
+	public void assertNotIntrumented_should_accept_non_jacoco_memebers() {
 		InstrSupport.assertNotInstrumented("run", "Foo");
 	}
 
 	@Test(expected = IllegalStateException.class)
-	public void testAssertNotIntrumentedField() {
+	public void assertNotIntrumented_should_throw_exception_when_jacoco_data_field_is_present() {
 		InstrSupport.assertNotInstrumented("$jacocoData", "Foo");
```

Comment:
I think it worth to add assertion for message especially since its change originates from a user needs. In past we have examples of catching regressions thanks to assertions for error messages.